src/betterproto/grpc/grpclib_server.py

```diff
@@ -0,0 +1,40 @@
+from abc import ABC
+from collections import AsyncIterable
+from typing import Callable, Any, Dict
+
+import grpclib
+import grpclib.server
+
+
+class ServiceBase(ABC):
+    """
+    Base class for async gRPC servers.
+    """
+
+    async def _call_rpc_handler_server_unary(
+        self,
+        handler: Callable,
+        stream: grpclib.server.Stream,
+        request_kwargs: Dict[str, Any],
+    ) -> None:
+
+        response = await handler(**request_kwargs)
```

Comment:
this logic seems simple enough that you might as well just inline it in the generate code?

Reply:
I did this for consistency. But yeah, I suppose 5 lines of code + an abstract method is a bit excessive for something that's a one-liner.